### PR TITLE
adds qt-faststart on ffmpeg build

### PR DESCRIPTION
**Pull request by vitalis**
a very useful tool for us

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -159,3 +162,4 @@ yum remove -y autoconf automake gcc gcc-c++ git libtool nasm  zlib-devel tar xz
 yum clean all
 rm -rf /var/lib/yum/yumdb/*
 echo "/usr/local/lib" > /etc/ld.so.conf.d/libc.conf
+mkdir -p /mnt/tmp
```
Comment (jrottenberg) [inline run.sh]: What is it for ?

The Dockerfile uses :

 WORKDIR /tmp/workdir

Assuming all the files to be managed would be stored there

File: run.sh [hunk 2]
```diff
@@ -143,6 +143,9 @@ DIR=$(mktemp -d) && cd ${DIR} && \
               make install && \
               make distclean && \
               hash -r && \
+              cd tools
+              make qt-faststart
+              cp qt-faststart ${SRC}/bin
```
Comment (jrottenberg) [inline run.sh]: oh I've never used it, pretty cool, thx.

File: run.sh (hunk 1 above)
Comment (vitalis) [inline run.sh]: added by mistake, i'll issue a new pull request

**Comment by jrottenberg** [comment]
It's too bad there is not target in the Makefile to build all those cool tools and installed them, but if you guys need qt-faststart for now, that's cool.

**Pull request merged**